src/main/java/lotto/view/ResultView.java

```diff
@@ -14,16 +11,16 @@
 
     private static final DecimalFormat FORMAT = new DecimalFormat("#.##");
 
-    public static void outputPurchaseQuantity(int quantity) {
-        System.out.println(quantity + "개를 구매했습니다.");
+    public static void outputPurchaseQuantity(LottoTicket ticket) {
+        System.out.println("수동으로 " + ticket.getManualCount() + "장, 자동으로 " + ticket.getAutoCount() + "개를 구매했습니다.");
     }
 
     public static void outputLottos(Lottos lottos) {
         IntStream.range(0, lottos.quantity())
                 .forEach(i -> System.out.println(lottos.getLotto(i).stream()
-                                                            .map(lottonumber -> lottonumber.getNumber())
-                                                            .sorted()
-                                                            .collect(Collectors.toList())));
+                        .map(lottonumber -> lottonumber.getNumber())
```

Comment:
```suggestion
                        .map(LottoNumber::getNumber)
```
매서드 레퍼런스 사용을 통해 가독성을 높여볼 수 있을거 같아요! 😄